.github/workflows/test.yml

```diff
@@ -86,16 +86,21 @@ jobs:
       - name: Install Swift dependencies
         if: startsWith(matrix.os, 'macos')
         run: |
+          curl -OL https://swift.org/builds/swift-5.2.1-release/xcode/swift-5.2.1-RELEASE-oss.pkg
+          sudo installer -verbose -pkg swift-5.2.1-RELEASE-oss.pkg -target /
           brew update
           brew install mint
           cd ./test/linters/projects/swiftformat/
           mint bootstrap --link
           cd ../swiftlint/
           mint bootstrap --link
+          cd ../swift-format
+          brew install swift-format
 
       # Tests
 
       - name: Run tests
         run: |
+          export TOOLCHAINS=swift
```

Comment:
This is causing the Windows tests to fail. Could you please move the line into the Swift section?